components/image.css

```diff
@@ -0,0 +1,30 @@
+@import url(/stylesheets/main.css);
+
+.image-container{
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    padding: 1rem;
+}
+
+.image{
+    width: max-width;
```

Comment:
I think you're trying to give max-width to image. The syntax of max-width is: "max-width: value;".
 e.g., "max-width: 5rem;"

Reply:
made the necessary changes.